spec/lang/intrinsic.md

```diff
@@ -7,16 +7,29 @@ impl<M: Memory> Machine<M> {
         &mut self,
         intrinsic: Intrinsic,
         arguments: List<Value<M>>,
-    ) -> NdResult<(Value<M>, Type)> { .. }
+        ret_ty: Type,
+    ) -> NdResult<Value<M>> { .. }
 }
 ```
 
 This helper functions simplifies unit-returning intrinsics.
 
 ```rust
-fn unit<M: Memory>() -> (Value<M>, Type) {
-    (Value::Tuple(list![]), Type::Tuple{fields:list![], size: Size::ZERO})
+fn unit_value<M: Memory>() -> Value<M> {
+    Value::Tuple(list![])
 }
+
+fn unit_type() -> Type {
+    Type::Tuple{fields:list![], size: Size::ZERO}
```

Comment:
```suggestion
    Type::Tuple { fields: list![], size: Size::ZERO }
```
Let's follow standard Rust formatting

Reply:
Is this a single case or should I go over all my changes reviewing this?

Reply:
Ideally this should be consistent everywhere.